plugins/elasticsearch/unit_test/test_search_documents.py

```diff
@@ -121,3 +117,18 @@ def test_search_documents_wrong_object(self, mock_request):
             },
             actual,
         )
+
+    @patch("requests.request", side_effect=Util.mocked_requests_get)
+    def test_index_document_with_empty_response(self, mock_request):
+        with self.assertRaises(PluginException) as error:
+            self.action.run(
+                {Input.INDEX: "wrong_object", Input.QUERY: {"query": {"match_all": {}}}, Input.ROUTING: None}
+            )
+
+        self.assertEqual(error.exception.cause, "Wrong input 'query'.")
+        self.assertEqual(
+            error.exception.assistance,
+            "Old query style detected during input. Input shouldn't contain {'query': {'query': ...}}. Please refer "
+            "help for more details or elasticsearch API documentation: https://www.elastic.co/guide/en/elasticsearch/"
+            "reference/current/query-filter-context.html#query-filter-context-ex",
+        )
```

Comment:
```suggestion
        self.assertEqual(
            error.exception.assistance,
            "Old query style detected during input. The input shouldn't contain {'query': {'query': ...}}. "
                "Please refer to the help.md for more details or to the Elasticsearch API documentation: "
                "https://www.elastic.co/guide/en/elasticsearch/reference/current/query-filter-context.html#query-filter-context-ex",
        )
```

Reply:
This is a pretty brittle test. We don't necessarily need to test that the error output troubleshooting string is exactly the same. We could simply do `assertRaises(PluginException)`.